dishes/colombian/otras_preparaciones/envueltos.md

```
@@ -1,108 +1,33 @@
+
+## Preparación
+
+1.  Con cuidado, retira las hojas de las mazorcas, tratando de mantenerlas enteras. Lávalas y resérvalas.
+2.  Desgrana las mazorcas con un cuchillo.
+3.  Muele los granos de maíz en un procesador de alimentos o molino hasta obtener una masa suave.
+4.  En un tazón grande, mezcla la masa de maíz con la mantequilla derretida, el azúcar, la leche, el polvo de hornear y la sal.
+5.  Para armar los envueltos, toma dos hojas de mazorca y únelas por los bordes más anchos.
+6.  Vierte una porción de la masa en el centro de las hojas. Si deseas, agrega un cubo de queso en el centro.
+7.  Dobla los lados de las hojas hacia el centro y luego dobla las puntas para cerrar el envuelto. Puedes amarrarlo con tiras de las mismas hojas.
+8.  En una olla grande, coloca una base de tusas (el corazón de la mazorca) y un poco de agua.
+9.  Coloca los envueltos parados sobre las tusas, tapa la olla y cocina al vapor durante 45-60 minutos, o hasta que al insertar un palillo, este salga limpio.
```

markdownlint-cli2 (0.17.2)

Spaces after list markers
Expected: 1; Actual: 2

(MD030, list-marker-space)